### DISCUSSION TITLE
Cookie consent

### DISCUSSION THREAD

**Discussion by noxitechdk** [Paymenter/Paymenter#292] [Ideas]
For compliance with legal requirements on the internet, it is advisable to incorporate a cookie manager or a tool for collecting cookie consent.

**Comment by finnjosten** [comment]
very much depends, as long as all cookies used are purely for analytics (THAT ARENT SHARED) or the site to function its not needed for a user to consent the cookies. How ever if cookies are used for advertising and sharing and all that kinda stuff its indeed required to have a consent form. I still suggest adding something like it though
